L1Trigger/L1THGCal/interface/HGCalModuleDetId.h

```diff
@@ -33,12 +35,18 @@ class HGCalModuleDetId : public DetId {
   /** Create cellid from raw id (0=invalid tower id) */
   HGCalModuleDetId(uint32_t rawid);
   /** Constructor from subdetector, zplus, type, layer, sector, module numbers */
-  HGCalModuleDetId(ForwardSubdetector subdet, int zp, int type, int layer, int sector, int moduleU, int moduleV);
+  HGCalModuleDetId(int subdet, int zp, int type, int layer, int sector, int moduleU, int moduleV);
```

Comment:
Would be better to use `HGCalTriggerSubdetector subdet` instead of `int subdet`

Reply:
thanks, done